contrib/build-windows.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/sh
+set -e
+
+root=$(pwd)
+export DESTDIR=${root}/dist
+build=$root/build-win32
+mkdir -p "$build" && cd "$build"
+
+# install deps
+if [ ! -f /usr/share/mingw/toolchain-mingw64.meson ]; then
+    ./contrib/ci/fwupd_setup_helpers.py -v mingw64 install-dependencies
+fi
+
+# try to keep this and ../contrib/ci/build_windows.sh in sync as much as makes sense
+meson .. \
+    --cross-file=/usr/share/mingw/toolchain-mingw64.meson \
+    --prefix=/ \
+    --sysconfdir="etc" \
+    --libexecdir="bin" \
+    --bindir="bin" \
+    -Dbuild=all \
+    -Ddocs=none \
+    -Dhsi=false \
+    -Dman=false \
+    -Dfish_completion=false \
+    -Dbash_completion=false \
```

Comment:
what do you think about converting these two to tristate options that auto-disable for os == windows?

Reply:
do either *ever* make sense on Windows? If not, then yes please, although I think you're the expert at the auto-disable magic.

Reply:
The only time you run bash in Windows is WSL, and if you're in WSL you won't read bash-completion.  So I don't think they ever make sense in Windows